docs/Configuring Banned Methods.md

```diff
@@ -0,0 +1,46 @@
+# Configuring Banned Methods (NI1006)
+
+NI1006 is an analyzer rule that prevents specific methods from being used in source. This ensures developers don't accidentally introduce hard-to-find bugs, inappropriate diagnostics, obsolete API usages, etc.
+
+# Process
+
+The analyzer works by looking through "BannedMethods" [additional files](https://github.com/dotnet/roslyn/blob/master/docs/analyzers/Using%20Additional%20Files.md). A project can contain any number of files as long as:
```

Comment:
"This analyzer works by loading banned methods from [additional files](url). An additional file's contents will be parsed if the following conditions are true:"

Reply:
Nice, thanks! Updated.